Hi @atoulme  - i am getting an issue while running the make command 

```
$ make update-codeowners
cd cmd/githubgen && go install .
githubgen
2023/10/26 14:06:13 codeowners are not members: BinaryFissionGames, Frapschen, JonathanWamsley, MovieStoreGuy, Mrod1598, StefanKurek, VihasMakwana, adrielp, ag-ramachandran, albertteoh, alexvanboxel, altuner, antonblock, armstrmi, atingchen, atoulme, avadhut123pisal, bertysentry, bryan-aguilar, chrroberts-pure, codeboten, cparkins, cpheps, damemi, dashpole, dehaansa, dgoscn, erichsueh3, evan-bradley, fatsheep9146, fchikwekwe, gjanco, hickeyma, hughesjj, jacobmarble, jamesmoessis, jmacd, jsuereth, kentquirk, khyatigandhi6, kongluoxing, liustanley, mapno, mar4uk, martin-majlis-s1, matej-g, mcardy, mrsillydog, nslaughter, pavankrish123, pcwiese, pjanotti, portertech, povilasv, psx95, pxaws, qiansheng91, rapphil, rmfitzpatrick, rnishtala-sumo, rogercoll, samiura, schmikei, shabicheng, shalper2, srprash, tamir-michaeli, tomaz-s1, varunraiko, wangzlei, yotamloe, zdaratom
make: *** [gengithub] Error 1

```

--- a/.github/CODEOWNERS
+++ b/.github/CODEOWNERS
@@ -90,7 +90,7 @@ exporter/zipkinexporter/                                                @open-te
 extension/asapauthextension/                                            @open-telemetry/collector-contrib-approvers @jamesmoessis @MovieStoreGuy
 extension/awsproxy/                                                     @open-telemetry/collector-contrib-approvers @Aneurysm9 @mxiamxia
 extension/basicauthextension/                                           @open-telemetry/collector-contrib-approvers @jpkrohling @svrakitin @frzifus
-extension/bearertokenauthextension/                                     @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123 @frzifus
+extension/bearertokenauthextension/                                     @open-telemetry/collector-contrib-approvers @jpkrohling @frzifus
 extension/encoding/jaegerencodingextension/                             @open-telemetry/collector-contrib-approvers @MovieStoreGuy @atoulme
 extension/encoding/jsonlogencodingextension/                            @open-telemetry/collector-contrib-approvers @VihasMakwana @atoulme
 extension/encoding/textencodingextension/                               @open-telemetry/collector-contrib-approvers @MovieStoreGuy @atoulme

